index.html

```diff
@@ -16,11 +19,36 @@
 
 <body>
 
-  
+    <header>
+        <div>
+          <div class="logo">
+          <h1>Fun Bus</h1>
+        </div>
+          <nav>
+            <a href="#">Home</a>
+            <a href="#">About Us</a>
+            <a href="#">Blog</a>
+            <a href="#">Contact</a>
+          </nav>
+        </div>
+    </header>
+      <div class="container home">
```

Comment:
You have some major issues with your html.  I changed the html and the view is mostly fixed.

The nav header section is fine, but the top header section is off by 2 elements.

You have dual <div container home> and you're also missing a closing section tag

Reply:
the first clue I had about the html not being correct was that it wasn't formatting.  So next time if it isn't formatting, something is probably wrong